yarn.lock

```diff
@@ -0,0 +1,506 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We do not use yarn here, please check out the contributing guide, on how to use the repository infra :) https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i